tests/queries/0_stateless/03255_fix_sbstrings_logical_error.sql

```diff
@@ -0,0 +1,9 @@
+SELECT 
```

Comment:
It would be nice to use _minimal_ reproducible examples as test cases. Otherwise, someone who tries to find out which arguments actually crashed `countSubstrings` will have a hard time.